partitura/io/importmusicxml.py

```diff
@@ -187,6 +190,11 @@ def load_musicxml(xml, ensure_list=False, validate=False, force_note_ids=None):
         A list of either Part or PartGroup objects
 
     """
+    
+    if os.path.basename(xml).endswith(".mxl"):
+        with zipfile.ZipFile(xml) as zipped_xml:
+            xml = zipped_xml.open(os.path.basename(xml).split(".")[0]+".xml")
```

Comment:
And the same here (since "." can be part of the file name and not be necessarily the extension)